lambda/rotator-snowflake/main.go

```diff
@@ -9,23 +9,26 @@ import (
 	"github.com/chanzuckerberg/go-misc/lambda/rotator-snowflake/rotate"
 	"github.com/chanzuckerberg/go-misc/lambda/rotator-snowflake/setup"
 	"github.com/hashicorp/go-multierror"
+	"github.com/segmentio/chamber/store"
 	"github.com/sirupsen/logrus"
 )
 
 var localFlag = flag.Bool("local", false, "Whether this lambda should be run locally")
 
 func Rotate(ctx context.Context) error {
-	databricksAccount, err := setup.Databricks(ctx)
+	secretStore := store.NewSSMStore(2)
```

Comment:
Instead of using 2 magic number, would you mind making it a named var to contextualize it?